### ISSUE TITLE
Allow customize the CloudWatch scheduled name

### ISSUE THREAD

**Issue by hoang-innomizetech** [awslabs/serverless-application-model#934]
Currently, Schedule event type only supports two props, I think it would be better if we can control more props of the CloudWatch event likes below

```yaml
Type: Schedule
Properties:
    Name: scheduled name
    Description: description
    Enabled: true/false
    Schedule: rate(1 hour)
    Input: ''
```

We have faced an issue when updating the scheduled event on CloudWatch **RE: Member must have length less than or equal to 100.**

**Comment by keetonian** [comment]
That is interesting that the length can be greater than 100. Looking at CW schedule events, the naming schema is {LambdaFunctionName}{EventName}.

It looks like the new proposed properties are `Enabled`, `Name`, and `Description`. 

Those properties can be added to the Schedule event here: https://github.com/awslabs/serverless-application-model/blob/master/samtranslator/model/eventsources/push.py#L69

Then update relevant tests to test these new properties.

**Comment by MartinDevillers** [comment]
Voted +1 👍 
I have a lambda function with 4 different scheduled events. Inlining the events inside the Serverless Function means I can no longer specify the name and the auto-generated names don't include the identifier of the event. Thus it becomes very hard to distinguish between the 4 events inside AWS.

**Comment by mcintyre94** [comment]
Being able to control the `enabled` property here would be really nice too - it'd be good to be able to deploy while testing with schedules disabled instead of having to go and set them to crons in the past to make sure they don't run.

**Comment by jlhood** [comment]
Pending v1.14.0 release

**Comment by ShreyaGangishetty** [comment]
Closing this issue as this is released in v1.14.0
